img.js

```diff
@@ -284,6 +309,16 @@ async function resizeImage(src, options = {}) {
   let fullStats = getFullStats(src, metadata, options);
   for(let outputFormat in fullStats) {
     for(let stat of fullStats[outputFormat]) {
+      if(options.useCache && fs.existsSync(stat.outputPath)){
+        stat.size = fs.statSync(stat.outputPath).size;
+        if(options.dryRun) {
+          stat.buffer = fs.readFileSync(src);
```

Comment:
another read (see comment above)

Reply:
This read happens only in dryRun, but I don't know how to circumvent calling `fs.readFileSync`.. since `stat.buffer` [needs to have the file content](https://github.com/11ty/eleventy-img/blob/master/test/test.js#L491)

Reply:
(going to continue the conversation on the other thread)